notebookUtils.py

```diff
@@ -60,15 +60,21 @@ def saveFile(file_path, token, log):
         return r.json()
 
 
-def updateApi(token):
+def updateApi(token, log):
     """
     Update autosave timestamp in Sandboxes API
     """
 
-    headers = {'X-StorageApi-Token': token, 'User-Agent': 'Keboola Sandbox Autosave Request'}
+    headers = {
+        'Content-Type': 'application/json',
+        'User-Agent': 'Keboola Sandbox Autosave Request',
+        'X-StorageApi-Token': token,
+    }
     url = os.environ['SANDBOXES_API_URL'] + '/sandboxes/' + os.environ['SANDBOX_ID']
-    json = {'lastAutosaveTimestamp': datetime.now().strftime('%Y-%m-%dT%H:%M:%S.%f')}
-    retrySession().put(url, json, headers=headers)
```

Comment:
This `json` parameter was the problem, the requests lib did not translate it correctly. I also added logging of API errors.